app/src/main/res/drawable/message_outgoing.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/design_default_color_primary" />
+    <corners android:topRightRadius="5dp" android:radius="15dp" />
```

Comment:
can the corner dimensions be moved into `dimens.xml` and shared by `message_incoming` and `message_outgoing`?